magellan-legacy/src/main/java/com/wealthfront/magellan/Screen.java

```diff
@@ -243,49 +217,31 @@ protected final void setTitle(CharSequence title) {
    * on rotation.
    */
   protected final void showDialog(DialogCreator dialogCreator) {
-    this.dialogCreator = dialogCreator;
-    this.dialogIsShowing = true;
-    createDialog();
-  }
-
-  /**
-   * Override this method to implement a custom behavior one back pressed.
-   *
-   * @return true if the method consumed the back event, false otherwise.
-   */
-  @Override
-  public boolean handleBack() {
-    return false;
+    dialogComponent.showDialog(dialogCreator);
   }
 
   /**
    * @return a String representation of the Screen to be used for logging purposes. Return the Simple name of the class
    * by default.
    */
+  @NonNull
   @Override
   public String toString() {
     return getClass().getSimpleName();
   }
 
-  @VisibleForTesting
   public final void setView(V view) {
     this.view = view;
   }
 
-  @VisibleForTesting
   public final void setActivity(Activity activity) {
     this.activity = activity;
   }
 
-  @VisibleForTesting
-  public final void setNavigator(Navigator navigator) {
+  public void setNavigator(@NotNull LegacyNavigator navigator) {
```

Comment:
`internal`? Or `@VisibleForTesting`?

Reply:
internal does exist in java unfortunately. Changed to package-private.